SharpPcap/LibPcap/SendQueue.cs

```diff
@@ -111,9 +111,27 @@ public bool Add(PcapHeader header, byte[] packet)
         /// Should the timestamps be respected
         /// </param>
         /// <returns>
-        /// A <see cref="int"/>
+        /// The number of bytes sent as an <see cref="int"/>
         /// </returns>
         public int Transmit(PcapDevice device, bool synchronized)
+        {
+            return Transmit(device, (synchronized == true) ? SendQueueTransmitModes.Synchronized : SendQueueTransmitModes.Normal);
+        }
+
+        /// <summary>
+        /// Send a queue of raw packets to the network.
+        /// </summary>
+        /// <param name="device">
+        /// The device on which to send the queue
+        /// A <see cref="PcapDevice"/>
+        /// </param>
+        /// <param name="transmitMode">
+        /// Should the timestamps be respected
+        /// </param>
+        /// <returns>
+        /// The number of bytes sent as an <see cref="int"/>
+        /// </returns>
+        public int Transmit(PcapDevice device, SendQueueTransmitModes transmitMode)
```

Comment:
There is no point introducing an enum that's basically the same as true/ false

Reply:
@kayoub5 consider the usage. Previously it was Transmit(someDevice, true); what does that mean? The user ends up having to look at the signature for the call to decode it. With the enum the code becomes Transmit(someDevice, synchronous);. Imo its always helpful to use an enum rather than a boolean for this reason.